argocd-boostrap/argobootstrap-job.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: test
+spec:
+  template:
+    spec:
+      containers:
+      - name: pi
+        image: andwizzle93/argocd-bootstrap
```

Comment:
Let's add an explicit tag that matches the version of the argocd client. Applies to docker build and ECR as well.

Reply:
Added a tag to address this